freqtrade/commands/cli_options.py

```diff
@@ -614,4 +614,36 @@ def __init__(self, *args, **kwargs):
               "that do not contain any parameters."),
         action="store_true",
     ),
+    "analysis_groups": Arg(
+        "--analysis_groups",
```

Comment:
Please use dash (`-`) in commands (so the above would become `--analysis-groups`).
That'll keep commandline-arguments aligned thorughout freqtrade - otherwise they're too easy to miss-type (if you need to also remember _ or -.)

Reply:
Done